README.md

```diff
@@ -129,3 +129,161 @@ Then just run `./build.pl`.
 The above command, run on a CentOS 7 machine with the options as shown in `config.build`, created the following:
 
     $HOME/installer-build/BUILDS/RHEL7_64/JUDASPRIEST-876/20170323061131_FOSS/zm-build/zcs-8.7.6_GA_1713.RHEL7_64.20170323061131.tgz
+
+# Development
+
+## Setup
+
+The following is a walk-through of the basic steps required to do ZCS development.  The first step is to simply install a current FOSS build on the machine that you wish to use.  The instructions that follow assume that this has been done.
+
+
+1. Create `/home/zimbra` and make `zimbra` the owner.
+
+		sudo mkdir /home/zimbra
+		sudo chown zimbra:zimbra /home/zimbra
+
+2. Install  `git`, `ant`, and `ant-contrib`.
+
+		sudo apt-get install git and ant-contrib
```

Comment:
Thank you! I noticed that and two other errors or omissions right after submitting the pull request.  Please refresh.